packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -795,6 +796,138 @@ describe("Integration tests", () => {
         );
       });
     });
+    describe("internalRevokeCertifiedAttribute", async () => {
+      const correlationId = generateId();
+      const requesterTenant: Tenant = {
+        ...mockTenant,
+        features: [
+          {
+            type: "PersistentCertifier",
+            certifierId: generateId(),
+          },
+        ],
+        updatedAt: currentDate,
+        name: "A requesterTenant",
+        externalId: {
+          origin: generateId(),
+          value: "1234567",
+        },
+      };
+      const attribute: Attribute = {
+        name: "an Attribute",
+        id: unsafeBrandId(requesterTenant.id),
+        kind: "Certified",
+        description: "an attribute",
+        creationTime: new Date(),
+        code: "123456",
+        origin: requesterTenant.externalId.origin,
+      };
```

Comment:
```suggestion
```
This can be a `getMockAttribute`, the id is not the one of the tenant, but the specific id for this specific attribute, origin too is not tied to the origin of the tenant